docfx/doxygen2markdown.cc

```diff
@@ -552,6 +552,9 @@ bool AppendIfProgramListing(std::ostream& os, MarkdownContext const& ctx,
   if (std::string_view{node.name()} != "programlisting") return false;
   // Start with a new paragraph, with the right level of indentation, and a new
   // code fence.
+  if (ctx.paragraph_start.empty()) {
+    os << "\n\n";
+  }
```

Comment:
Wouldn't it be nicer to change where `paragraph_start` was set to empty and make it `"\n\n"` instead?